src/index.js

```diff
@@ -1449,6 +1482,23 @@ module.exports = {
       }
     );
 
+    expect.exportAssertion(
+      '<DOMDocument|DOMElement|DOMDocumentFragment> [not] to have test id <string>',
+      (expect, subject, testId) => {
+        expect.errorMode = 'nested';
+        expect.subjectOutput = output =>
+          expect.inspect(subject, Infinity, output);
```

Comment:
Is this to avoid dotting out? Is it something we want in some of the other assertions as well?

Reply:
Yes, I have experience cases where we dot out stuff that contains conflicts. We should go through all of the assertions and make sure that we don't remove useful information, but that is a separate task: #287